users/views.py

```diff
@@ -1,3 +1,80 @@
-from django.shortcuts import render
+import json, re, bcrypt, jwt
 
-# Create your views here.
+from django.http import JsonResponse
+from django.views import View
+from django.conf import settings
```

Comment:
1. import 라인 맞춰주세요.
2. bcrypt와 jwt는 pip 설치한 외부 모듈이므로 두번째 블럭에 위치하게 해주세요.
```suggestion
import bcrypt
import jwt
from django.http  import JsonResponse
from django.views import View
from django.conf  import settings
```